save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/service/DockerService.kt

```diff
@@ -199,7 +197,11 @@ class DockerService(
             configProperties.testResources.basePath,
             execution.resourcesRootPath!!,
         )
-        val resourcesForExecution = createTempDirectory(prefix = "save-execution-${execution.id}")
+        val resourcesForExecution = createTempDirectory(
+            directory = Paths.get(configProperties.testResources.tmpPath),
+            prefix = "save-execution-${execution.id}"
+        )
+        log.debug { "Copying resources from $originalResourcesPath into $resourcesForExecution" }
         originalResourcesPath.copyRecursively(resourcesForExecution.toFile())
```

Comment:
btw, we create a tmp folder here and then create a new tmp folder in KubernetesPersistentVolumeService (which is not clear).
do we need two temp folders?

Reply:
I'll change this is #977. The logic will be like in Docker: create a tmp folder on a mount that can be shared with a temporary container, copy data from original storage to the tmp folder, copy from the tmp folder into a volume created for a certain execution,